python/cudf/cudf/core/series.py

```diff
@@ -1404,7 +1404,19 @@ def __repr__(self):
             cudf.core.column.timedelta.TimeDeltaColumn,
         ):
             output = repr(
-                preprocess.astype("O").fillna(cudf._NA_REP).to_pandas()
+                preprocess.astype("O")
```

Comment:
Is this affected by #13862?
```suggestion
                preprocess.astype("str")
```